doc/manual/generate-manpage.nix

```
@@ -107,8 +109,12 @@ let
 
                 ${option.description}
             '';
-          categories = sort lessThan (unique (map (cmd: cmd.category) (attrValues allOptions)));
-        in concatStrings (map showCategory categories);
+          categories = mapAttrs
+            (_: listToAttrs)
```

This confuses me. Why does the function that mapAttrs applies return listToAttrs?

---

The undoes the `attrsToList` further below.

We have:

- `allOptions : AttrSet<Option>`
- `attrsToList allOptions : List<{ name: String, value: Option }>`
- `groupBy ... : AttrSet<List<{ name: String, value: Option }>>`
- `mapAtrs ... : AttrSet<AttrSet<Option>>`

---

Ah yes, this is subtle. Maybe
```suggestion
            (_: listToAttrs) # undo `attrsToList allOptions` for each group
```

---

```suggestion
            # Convert each group from a list of key-value pairs back to an attrset
            (_: listToAttrs)
```